go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/gnolang/gno
 go 1.20
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220111032746-97732e52810c
-	github.com/btcsuite/btcd/btcutil v1.0.0
+	github.com/btcsuite/btcd/btcutil v1.1.0
```

Comment:
also, why not latest? (1.1.3)

Reply:
I have focused on updating btcec, not btcutil. This update was decided by go mod itself after updating to v2. Maybe the only version compatible with btcec v2 and our code?

Reply:
updated btcutil too.